specs/toolsets/adr_base/setup-toolset.sh

```diff
@@ -128,56 +100,24 @@ case $ADR_CPU_TYPE in
 esac
 
 export ADR_GCC_OPT=-O2
-export ADR_DIR_NDK_USR="${ADR_DIR_NDK}/platforms/$ADR_NDK_PLATFORM/arch-${ADR_CPU_TYPE}/usr"
 
 export ADR_SDK_PLATFORM_DIR="${ADR_DIR_SDK}/platforms/${ADR_SDK_PLATFORM}"
 if [ ! -e "$ADR_SDK_PLATFORM_DIR"  ]; then
     echo "E/Android toolset: can't find SDK platform:" ${ADR_SDK_PLATFORM_DIR}
     return 1
 fi
 
-export ADR_NDK_PLATFORM_DIR="${ADR_DIR_NDK}/platforms/${ADR_NDK_PLATFORM}"
-if [ ! -e "$ADR_NDK_PLATFORM_DIR"  ]; then
-    echo "E/Android toolset: can't find NDK platform:" ${ADR_NDK_PLATFORM_DIR}
-    return 1
-fi
-if [ ! -e "$ADR_DIR_NDK_USR"  ]; then
-    echo "E/Android toolset: can't find NDK platform for CPU arch:" ${ADR_DIR_NDK_USR}
-    return 1
-fi
 
-export ADR_SYSTEM_LINKLIBS=""
+export PATH=${PATH}:"${HAM_TOOLSET_DIR}":"${ADR_DIR_BASE}/scripts":"${ADR_DIR_SDK}/tools":"${ADR_DIR_SDK}/platform-tools"
```

Comment:
This should be updated to use pathenv_add, which is cleaner and prevents duplicate entries.

For example in the Python3 toolset:
```
        pathenv_add ${PYTHON3_DIR}/Scripts
        pathenv_add ${PYTHON3_DIR}/DLLs
        pathenv_add ${PYTHON3_DIR}
```